diff --git a/exercises/012_while2.zig b/exercises/012_while2.zig
index c9905aa..c878c67 100644
--- a/exercises/012_while2.zig
+++ b/exercises/012_while2.zig
@@ -25,7 +25,7 @@ pub fn main() void {
 
     // Please set the continue expression so that we get the desired
     // results in the print statement below.
-    while (n < 1000) : ??? {
+    while (n < 1024) : (n *= 2) {
         // Print the current number
         std.debug.print("{} ", .{n});
     }
